jina/peapods/pods/__init__.py

```diff
@@ -18,7 +18,27 @@
 
 
 class BasePod(ExitStack):
```

Comment:
I think the `BasePod` interface should be richer. At the end, it should be transparent to the `Flow` which concrete Pod is instantiating, therefore, every method needed by `Flow` to talk to `Pods` should somehow be thought as part as this interface.